src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
The name of the file or the factory name should match.  This should probably be `charts-mixin.factory.js` to match the established pattern.